cookbook/case_studies/ml_training/house_price_prediction/multiregion_house_price_predictor.py

```diff
@@ -1,17 +1,23 @@
 """
-
 Predicting House Price in Multiple Regions Using XGBoost and Dynamic Workflows
 -------------------------------------------------------------------------------
+`XGBoost <https://xgboost.readthedocs.io/en/latest/>`__ is an optimized distributed gradient boosting library designed to be efficient, flexible, and portable. 
+It uses `gradient boosting <https://en.wikipedia.org/wiki/Gradient_boosting>`__ technique to implement Machine Learning algorithms.
```

Comment:
Isn't this repetitive? We already have this in the other file (I'm sure I mentioned this earlier too).